sunbird_service_api_test/src/test/resources/templates/user/create/invalidChannelResponse.json

```diff
@@ -0,0 +1,15 @@
+{
+"id":"api.user.create",
+"ver":"v1",
+"ts":"@ignore@",
+"params":
+{
+"resmsgid":null,
+"msgid":"@ignore@",
+"err":"INVALID_PARAMETER_VALUE",
+"status":"INVALID_PARAMETER_VALUE",
+"errmsg":"Invalid value invalidChannel123123 for parameter channel. Please provide a valid value."
```

Comment:
make  invalidChannel123123 as more unique , so that it won't match with any existing channel value.